sdk/core/azure-core/src/main/java/com/azure/core/implementation/serializer/HttpResponseBodyDecoder.java

```diff
@@ -399,7 +402,8 @@ private static Type extractEntityTypeFromReturnType(HttpResponseDecodeData decod
                     // Get Type of 'T' from OperationStatus<T>
                     token = TypeUtil.getTypeArgument(token);
                 }
-            } catch (Exception ignored) {
+            } catch (ClassNotFoundException ignored) {
+                logger.warning("Failed to find class 'com.azure.core.management.implementation.OperationStatus'.");
```

Comment:
This should log and throw. ClassNotFoundException means the necessary libraries are not loaded at runtime. Instead of logging and ignoring it, we should propagate this exception upwards.

Reply:
@anuchandy Could you give me some insights on whether we need to throw exception or silence pass when the method encountered   ClassNotFoundException ?